packages/cf-style-const/src/variables.js

```diff
@@ -14,6 +14,20 @@ export default {
     seven: 'linear-gradient(left, #F69259, #FFDB6F)',
     eight: 'linear-gradient(left, #85CBA8, #FFDB6F)'
   },
+  color: {
+    rain: '#408BC9',
```

Comment:
I would rather rename these something that indicates the color.  We we need to map the design names to our names we should comment `sky` etc next to the hex code.  Even if we do something like `blueSky`, `redDawn` etc.